app/src/main/res/values/settings_keys.xml

```diff
@@ -55,6 +55,25 @@
         <item>30000</item>
     </string-array>
 
+    <string name="progressive_load_interval_key">progressive_load_interval</string>
+    <string name="progressive_load_interval_bytes_default_value">16384</string>
+    <string-array name="progressive_load_interval_descriptions">
+        <item>1 KiB</item>
+        <item>4 KiB</item>
+        <item>16 KiB</item>
+        <item>64 KiB</item>
+        <item>256 KiB</item>
```

Comment:
I'm not sure if all those entries are useful.
Is there so much difference between 1 and 4 kB?
Usually network connections nowadays have at least 1MB/s and an under limit of ~8-64kB/s.

I think a list containing
* use default
* 1kB
* 16kB
* 256kB

would be enough.

Reply:
>  Is there so much difference between 1 and 4 kB?
Nope. At least on my device, there is no difference between 1/4/16 kB, but I do notice a very small difference going from 16 to 64. Will remove the 1kB option for now.